data/topics.yml

```diff
@@ -142,6 +142,8 @@
       url: "troubleshooting/app-processing-requests-slowly"
     - title: "The connection to my MySQL database is being refused, what should I do?"
       url: "troubleshooting/mysql-connection-refused"
+    - title: "What are support request best practices?"
+      url: "troubleshooting/support-ticket"
```

Comment:
Let's get the slug to be closer to the question, to really juice the most out of SEO.

```
url: troubleshooting/support-ticket-best-practices
```

Also, the more verbose title "What are the best practices when opening support tickets?" may be clearer.